analysis/config.py

```diff
@@ -1,3 +1,8 @@
+# Dataset definition: dataset_definition_tables.py
+# Timeframe: 31/01/2024 - 31/01/2025
+start_date_tables = "2024-01-31"
+index_date = "2025-01-31"
```

Comment:
I'd follow the same naming convention here that we already have for the measures

```suggestion
start_date_dataset_tables = "2024-01-31"
index_date_dataset_tables = "2025-01-31"
```